uber/templates/static_views/prereg_not_yet_open.html

```diff
@@ -13,7 +13,7 @@ <h1 style='color:red'>{{ c.EVENT_NAME_AND_YEAR }} pre-registration is not yet op
   {%- if c.DEALER_REG_START and c.DEALER_REG_PUBLIC and c.BEFORE_DEALER_REG_SHUTDOWN -%}
   <p>
     <strong>Dealers:</strong>
-    {% if c.BEFORE_DEALER_REG_PUBLIC %}
+    {% if c.BEFORE_DEALER_REG_START %}
```

Comment:
Huh, that's a good point.

I'm the one who added dealer_reg_public, and while MAGFest doesn't use it, the idea was that it would sometimes be set to after dealer_reg_start so that dealer reg could be 'soft launched' before we show the link. But as you mentioned, the text will display weirdly. The correct thing to do here then is either:

1. check for the truthiness of AFTER dealer_reg_public, not the truthiness of whether the date is set at all
2. display the reg start time as dealer_reg_public, not dealer_reg_start

That's a minor concern, though.